README.md

```diff
@@ -196,6 +196,10 @@ The following environment variables can be used to configure Livebook on boot:
   * `LIVEBOOK_BASE_URL_PATH` - sets the base url path the web application is
     served on. Useful when deploying behind a reverse proxy.
 
+  * `LIVEBOOK_PUBLIC_BASE_URL_PATH_OVERRIDE` - overrides the base url path for the public
+    asset routes only. Useful to create exceptions when deploying behind a reverse proxy 
+    that requires authentication.
```

Comment:
```suggestion
  * `LIVEBOOK_PUBLIC_BASE_URL_PATH` - sets the base url path the `/public/*` routes
    are served on. Note that this takes precedence over `LIVEBOOK_BASE_URL_PATH`,
    if both are set. Setting this may be useful to create exceptions when deploying
    behind a reverse proxy that requires authentication.
```

Reply:
Done :)